### PR TITLE
Performance Improvement in the Handbook section

### LINKED ISSUE #3014: [Performance] Image Optimization <img> to <StaticImages> in the handbook section
**Current Behavior**

The .png and .jpg images cause a lot of performance issues 
**Desired Situation**

Change `` to `<StaticImagess>` for image optimization in the handbook section.  eg [PR](https://github.com/layer5io/layer5/pull/2597)  [StaticImagePlugin](https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/)
---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by legendarykamal**
**Description**

This PR fixes #3014 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit db4d4d652a06382a01e6eda24be7f368d16483b3 at: https://62f66ef0a954bc032cc98811--layer5.netlify.app

File: src/sections/Community/Handbook/projects.js [hunk 1]
```diff
@@ -60,8 +60,8 @@ const Maintainer = () => {
 
               Its service mesh community represents the largest collection of
               service Mesh projects.Emerging projects like Nighthawk{" "}
-              <img className="project-description-icon" src={hawkIcon} /> , community with{" "}
-              <Link to="/community/meshmates"><img className="project-description-icon" src={meshmateicon} /> MeshMates </Link>, catch-all org, “home base”.
+              <img className="project-description-icon" src={hawkIcon} alt="hawkIcon" /> , community with{" "}
+              <Link to="/community/meshmates"><img className="project-description-icon" src={meshmateicon} alt="meshmateicon" /> MeshMates </Link>, catch-all org, “home base”.
```
Comment (debo19) [inline src/sections/Community/Handbook/projects.js:64]:
```suggestion
              <Link to="/community/meshmates"><img className="project-description-icon" src={meshmateicon} alt="MeshMate icon" /> MeshMates </Link>, catch-all org, “home base”.
```

File: src/sections/Community/Handbook/projects.js [hunk 2]
```diff
@@ -60,8 +60,8 @@ const Maintainer = () => {
 
               Its service mesh community represents the largest collection of
               service Mesh projects.Emerging projects like Nighthawk{" "}
-              <img className="project-description-icon" src={hawkIcon} /> , community with{" "}
-              <Link to="/community/meshmates"><img className="project-description-icon" src={meshmateicon} /> MeshMates </Link>, catch-all org, “home base”.
+              <img className="project-description-icon" src={hawkIcon} alt="hawkIcon" /> , community with{" "}
```
Comment (debo19) [inline src/sections/Community/Handbook/projects.js:63]:
```suggestion
              <img className="project-description-icon" src={hawkIcon} alt="Nighthawk Icon" /> , community with{" "}
```

File: src/sections/Community/Handbook/projects.js [hunk 3]
```diff
@@ -122,8 +122,8 @@ const Maintainer = () => {
             </p>
             <p>
               Meshery and its components Meshery Operator{" "}
-              <img className="project-description-icon" src={mesheryoperatoricon} /> and MeshSync{" "}
-              <img className="project-description-icon" src={mesherysyncicon} />
+              <img className="project-description-icon" src={mesheryoperatoricon} alt="mesheryoperatoricon" /> and MeshSync{" "}
```
Comment (debo19) [inline src/sections/Community/Handbook/projects.js:125]:
```suggestion
              <img className="project-description-icon" src={mesheryoperatoricon} alt="Meshery Operator Icon" /> and MeshSync{" "}
```

File: src/sections/Community/Handbook/projects.js [hunk 4]
```diff
@@ -122,8 +122,8 @@ const Maintainer = () => {
...
+              <img className="project-description-icon" src={mesherysyncicon} alt="mesherysyncicon" />
```
Comment (debo19) [inline src/sections/Community/Handbook/projects.js:126]:
```suggestion
              <img className="project-description-icon" src={mesherysyncicon} alt="MeshSync icon" />
```

File: src/sections/Community/Handbook/repository.js [hunk 5]
```diff
@@ -184,7 +184,7 @@ const Repository = () => {
                         <tbody key={project}>
                           <tr>
                             <td>
-                              <img className="site-icon inline" src={image} />&nbsp;{project}</td>
+                              <img className="site-icon inline" src={image} />&nbsp;{project} alt="project" </td>
```
Comment (debo19) [inline src/sections/Community/Handbook/repository.js:187]:
```suggestion
                              <img className="site-icon inline" alt="project" src={image} />&nbsp;{project} </td>
```

**Review by debo19 — CHANGES_REQUESTED** [review]

File: src/sections/Community/Handbook/repository.js (hunk 5 above)
Comment (legendarykamal) [reply src/sections/Community/Handbook/repository.js:187]: resolved :+1:

**Comment by l5io** [comment]
🚀 Preview for commit b6739159a6eccdaab9bb7d7cde10ea1a2e1e8876 at: https://62f793356b267d187a98d46f--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e694a43ca6e8cd7d2261c9e79e27e434cc62f04a at: https://6303461af0b2c9445f040bab--layer5.netlify.app

**Review by debo19 — COMMENTED** [review]
Screenshot 2022-08-23 at 5 59 06 PM
Please make this image full width.

**Comment by legendarykamal** [comment]
@debo19  done! I have alligned it full space considering the code images above it.
image

**Comment by legendarykamal** [comment]
@debo19  changes pushed :100:

**Comment by l5io** [comment]
🚀 Preview for commit c1b35fbf18c3e257d7beb565d22812f89266e1a1 at: https://630f0380f4a0b02e19e0189c--layer5.netlify.app

**Pull request merged**